website/docs/components/dropdown/partials/code/how-to-use.md

```diff
@@ -163,23 +109,84 @@ Note that `toggleText` is still required, because it supplies the `aria-label` f
+When using the `@href` argument, you’re indicating an external link (instead of a route). So, a few relevant HTML attributes are added–`target="_blank"` and `rel="noopener noreferrer"`. However, if the `@href` really _does_ point to an internal link or uses a different protocol (e.g., `mailto` or `ftp`), pass `@isHrefExternal={{false}}` to the component and it will not add any extra HTML attributes.
+!!!
+
+If you pass a `@href` argument, a link (`<a>` element) will be generated:
+
+```handlebars
+{{!-- The Doc::ListContainer component is just to help the component render properly --}}
+<Doc::ListContainer class="hds-dropdown-list">
+  <Hds::Dropdown::ListItem::Interactive @href="https://www.hashicorp.com/request-demo/terraform" @text="Request a demo" />
+</Doc::ListContainer>
+```
+
+#### Rendering a LinkTo (with `@route`)
+
+Pass a `@route` to render Ember `<LinkTo>`. If the route is external to your current engine, pass `@isRouteExternal={{true}}` to use `<LinkToExternal>` instead of `<LinkTo>`. For more details see the [Hds::Interactive component](/utilities/interactive/) documentation. All the standard arguments for the `<LinkTo/LinkToExternal>` components are supported (e.g., `models/model/query/current-when/replace`).
+
+```handlebars
+{{!-- The Doc::ListContainer component is just to help the component render properly --}}
+<Doc::ListContainer class="hds-dropdown-list">
+  <Hds::Dropdown::ListItem::Interactive @route="my.page.route" @model="my.page.model" @text="Activate cluster" />
+</Doc::ListContainer>
+```
+
+#### With a loading "interactive" item
+
+There may be use cases when it’s necessary to put an item in a "loading" state while the app performs some operations (e.g., asynchronously checking the user’s permissions to execute a certain operation once the Toggle’s been clicked).
+
+Pass the argument `@isLoading={{true}}` to the item. This will show a "loading" icon (even if an argument `@icon` is provided) and sets the item as non-interactive until the value of `@isLoading` is set to `false` again.
 
 ```handlebars
 <Hds::Dropdown as |dd|>
-  <dd.ToggleIcon @icon="settings" @text="settings menu" />
-  <dd.Title @text="Signed In" />
-  <dd.Description @text="email@domain.com" />
-  <dd.Separator />
-  <dd.Interactive @route="components" @text="Settings and Preferences" />
+  <dd.ToggleIcon @icon="more-horizontal" @text="Overflow Options" @hasChevron={{false}} />
+  <dd.Interactive @route="components" @isLoading={{true}} @text="Edit cluster" @color="action" @icon="edit" />
   <dd.Interactive @route="components" @text="Delete" @color="critical" @icon="trash" />
 </Hds::Dropdown>
 ```
+
+### ListItem::Generic
+
+!!! Info
+
+When using the “generic” ListItem, the product team is responsible for implementing the layout and accessibility.
+!!!
+
+`ListItem::Generic` allows you to pass custom elements to the Dropdown.
+
+<!-- TODO: Troubleshoot why this example isn't working correctly -->
```

Comment:
What's the issue you are having with it?

Reply:
It's not rendering the Generic block or anything after it, but it's also not throwing any errors. The contents within the Generic block are just another HDS component, so I'm probably just missing something. It was working in the old scrappy site.

Reply:
I can have a look

Reply:
@heatherlarsen I see it working in the deployed branch... 🤔
<img width="1126" alt="screenshot_2297" src="https://user-images.githubusercontent.com/686239/214954491-3cca90f7-d315-43fd-8e2d-57db32780049.png">
https://hds-website-git-heatherlarsen-component-dropdown-hashicorp.vercel.app/components/dropdown?tab=code#listitemgeneric

Reply:
Ohh, weird! It wasn't working for me last night, but glad it's ok. I'll remove the comment.